default/python/AI/CombatRatingsAI.py

```diff
@@ -216,6 +216,11 @@ def get_stats(self, hashable=False):
         """
         return self.get_basic_stats(hashable=hashable) + self.get_fighter_stats()
 
+    @property
+    def raw_attack(self):
```

Comment:
Little docstring will be helpfull.  

"""
Sum of all weapon damage without species bonuses.
"""